### ISSUE TITLE
AWS SNS `topicArn` cannot use a string input 

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/plugin-aws#178]
This flow will fail:

```yaml
id: sendSMS
namespace: blueprint
inputs:
  - name: smsText
    type: STRING
    defaults: "Hello from Kestra and AWS SNS!"
  - name: topicArn
    type: STRING
    defaults: arn:aws:sns:eu-central-1:338306982838:kestra
tasks:
  - id: sendSMS
    type: io.kestra.plugin.aws.sns.Publish
    region: eu-central-1
    accessKeyId: "{{envs.aws_access_key_id}}"
    secretKeyId: "{{envs.aws_secret_access_key}}"
    topicArn: "{{inputs.topicArn}}"
    from:
      data: |
        {{inputs.smsText}}
```

It's not possible to pass the input without wrapping it in quotes "" e.g. `topicArn: "{{inputs.topicArn}}"`

This results in an error: 

```
Invalid parameter: TopicArn Reason: An ARN must have at least 6 elements, not 1 (Service: Sns, Status Code: 400, Request ID: 774026a4-77cc-538e-ae87-7ba611efd03c) 
```

This is the format that works:

```yaml
id: sendSMS
namespace: blueprint

inputs:
  - name: smsText
    type: STRING
    defaults: "Hello from Kestra and AWS SNS!"

tasks:
  - id: sendSMS
    type: io.kestra.plugin.aws.sns.Publish
    region: eu-central-1
    accessKeyId: "{{envs.aws_access_key_id}}"
    secretKeyId: "{{envs.aws_secret_access_key}}"
    topicArn: arn:aws:sns:eu-central-1:338306982838:kestra
    from:
      data: |
        {{inputs.smsText}}
```

which means that `topicArn` expects an object making it impossible to pass it as a string even though the plugin documentation shows the attribute as a string:
![image](https://github.com/kestra-io/plugin-aws/assets/86264395/a651a765-27c0-4abe-bde5-27df222cc3cf)

## TL;DR

Add support for passing `topicArn`from the input
